packages/babel-preset/src/index.test.ts

```diff
@@ -13,11 +13,7 @@ pluginTester({
   babelOptions: {
     parserOpts: {
       plugins: ['typescript'],
-    },
-  },
-  pluginOptions: {
-    babelOptions: {
-      presets: ['@babel/typescript'],
+      sourceType: 'unambiguous',
```

Comment:
This will be probably defined in Linaria itself. Forces Babel to detect a type of module, so it will emit `import`/`require` properly.